deploy/charts/harvester/templates/rbac.yaml

```diff
@@ -201,4 +201,12 @@ rules:
   - volumes/status
   verbs:
   - get
-  - list
\ No newline at end of file
+  - list
+- apiGroups:
+  - snapshot.storage.k8s.io
+  resources:
+  - volumesnapshots
```

Comment:
should we need
- volumesnapshots/status
- volumesnapshotcontents/status
These two?